addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -582,7 +582,7 @@
     }
 
     .o_cw_filter_color_#{$i - 1} {
-        .o_cw_filter_input_bg {
+        .o_cw_filter_input_bg:not(.no_filter_color) {
```

Comment:
We should explain in the commit why we need to have this or we could also add it in planning instead.

Reply:
The fa-check icon that replace the avatar picture has the _o_cw_filter_input_bg_ class, but now one of it's parents has the o_cw_filter_color_{colorId} which wasn't the case previously their colorId was empty. Now that the fa-check has this color we need to make sure that there is a way to remove this background color or else the background of the check is colored as well.
I think it would be too heavy in term of code to cancel this from the planning calendar renderer, the class being inside of a css loop using specific colors constants. i'll explain it in the commit we'll see what they say

Reply:
Okay thanks for the explanation. 🙂